test/sql/visualizer/test_visualizer.test

```diff
@@ -0,0 +1,47 @@
+# name: test/sql/visualizer/test_visualizer.test
+# description: Profiler Visualizer
+# group: [visualizer]
+
+require visualizer
+
+statement ok
+PRAGMA enable_profiling='json'
+
+statement ok
+Pragma visualize_last_profiling_output ='__TEST_DIR__/test.html'
+
+statement ok
+PRAGMA profiling_output='__TEST_DIR__/first.json'
+
+statement ok
+PRAGMA profiling_mode = detailed
+
+statement ok
+CREATE TABLE integers(i INTEGER)
+
+statement ok
+CREATE TABLE t1 AS SELECT i as v1, i as v2 from range (0,10) t(i)
+
+statement ok
+CREATE TABLE t2 AS SELECT i as v1, i as v2 from range (0,5) t(i)
+
+statement ok
+SELECT t1.v1 FROM t1 JOIN t2 USING (v1)
+
+statement ok
+Pragma visualize_last_profiling_output ='__TEST_DIR__/test.html'
+
+statement ok
+SELECT t1.v1 FROM t1 JOIN t2 USING (v1)
+
+statement ok
+Pragma visualize_json_profiling_output('__TEST_DIR__/test.html','__TEST_DIR__/first.json')
+
+statement ok
+PRAGMA profiling_output='__TEST_DIR__/second.json'
+
+statement ok
+SELECT t1.v1 FROM t1 JOIN t2 USING (v1)
+
+statement ok
+Pragma visualize_diff_profiling_output('__TEST_DIR__/test.html','__TEST_DIR__/first.json','__TEST_DIR__/second.json')
```

Comment:
What if any of these files don't exist? Could you add a test with e.g. `statement error` to verify that it does not crash?

Reply:
> On that note, @azimafroozeh can you put a LICENSE file along with `d3.js` that contains the license of d3?
I think the first line of d3.js contains the LICENSE, no?